src/utils/index.coffee

```diff
@@ -121,11 +128,18 @@ createDirectoryCache = ({directory, recursive, extensions, allowed, context}) ->
         prefixRelativePath = "#{normalizePath dirName}#{name}"
         if 'filename' in allowed
           continue unless ext in extensions
-          cache.set name, {
-            prefixRelativePath
-            fullPath
-            type: 'filename'
-          }
+          cache.set(
+            if settings['known-imports/case-insensitive']
+              name.toLowerCase
```

Comment:
This looks like you need to actually call the `.toLowerCase()` method?

Reply:
Huh, how was this possibly working.